src/hCassandra_test.py

```diff
@@ -0,0 +1,181 @@
+            '.99': [],           # 99% of the time the latency was less than this number.
+            'max': [],           # Maximum latency in miliseconds.
+            'gc_num': 0,         # Number of garbage collections.
+            'max_ms': [],        # Longest garbage collection in miliseconds.
+            'sum_ms': 0,         # Total of gargage collection in miliseconds.
+            'sdv_ms': [],        # Standard deviation in miliseconds.
+            'mb': 0,             # Size of the garbage collection in megabytes.
+            'op_time': []        # Total Operation Time per client
+        }
+        cassandra_results = {
+            'write': dict(result),
+            'read': dict(result)
+        }
+        # Get stats for Cassandra Stress Client
+        stats = self.get_app_stats(self.stress_client)
+        # num_clients = self.options.total_client_count
+        db_ops = ['write', 'read']
+        for client in stats.keys():
+            info = stats[client]
+            # l.info(" CLIENT = " + pformat(client) + " DATA = " + pformat(info))
+            for db_op in db_ops:
+                if db_op in info:
+                    info[db_op] = ast.literal_eval(info[db_op])
+                    cassandra_results[db_op]['total ops'].append(info[db_op]['Total partitions'])
+                    cassandra_results[db_op]['op/s'].append(info[db_op]['op rate'])
+                    cassandra_results[db_op]['pk/s'].append(info[db_op]['partition rate'])
+                    cassandra_results[db_op]['.95'].append(info[db_op]['latency 95th percentile'])
+                    cassandra_results[db_op]['.99'].append(info[db_op]['latency 99th percentile'])
+                    cassandra_results[db_op]['gc_num'] += int(info[db_op]['total gc count'])
+                    cassandra_results[db_op]['sdv_ms'].append(info[db_op]['stdev gc time(ms)'])
+                    cassandra_results[db_op]['max'].append(info[db_op]['latency max'])
+                    cassandra_results[db_op]['med'].append(info[db_op]['latency median'])
+                    cassandra_results[db_op]['op_time'].append(info[db_op]['Total operation time'])
+
+        return cassandra_results
+
+    def launch_stress_client(self):
+        max_threads_per_client = 10
+        l.info("Launching the Cassandra Stress Client(s). Total clients = %s" % (self.options.total_client_count))
+        constraints = []    # no constraints defined for cassandra stress-clients
+        # Determine number of threads per Cassandra Stress Client
+        if self.options.total_client_count > max_threads_per_client:
+            threads_per_client = max_threads_per_client
+        else:
+            threads_per_client = self.options.total_client_count
+        l.debug("Number of Threads per Cassandra-Stress Client, set to: %s" % (threads_per_client))
+        self.create_hydra_app(name=self.stress_client, app_path='hydra.cassandra.stress_client.run',
```

Comment:
This will likely fail, you need to change it to "create_binary_app" and directly launch the "stress_client"
If' its not clear ping me back on chat.